sdk/servicebus/azure-messaging-servicebus/pom.xml

```diff
@@ -81,6 +81,12 @@
       <artifactId>azure-core-test</artifactId>
       <version>1.26.2</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>com.fasterxml.jackson.dataformat</groupId>
+          <artifactId>jackson-dataformat-xml</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
Do we need this exclusion since we are removing xml dependency from azure-core-test anyway? Next version update will not require this exclusion.

Reply:
I want to add the exclusion to make sure that when the next version of azure-core-test is released we don't have an unforeseen break in testing due to the removal.